### ISSUE TITLE
Check posterior_interval() is imported correctly

### ISSUE THREAD

**Issue by dvukcevic** [jeffreypullin/rater#76]
While testing #71, I found that the `posterior_interval()` generic didn't seem to be getting loaded/attached properly on my installation of R.

I am currently using an old version of R (version 3.4.4) and am the only one experiencing this problem.  Therefore, our current hypothesis is that it's due to my old version of R.

If this hypothesis is correct, we should identify which version of R is necessary for proper functioning and specify this as a dependency.

**Comment by jeffreypullin** [comment]
I think method registration was changed in R 3.6 but I'm not sure if that would have an impact here.

Another (perhaps unlikely) possibility is that you are using a version of rater before posterior intervals was included. To see whether this is the case I've just bumped the version to 0.1.9001. If you get the same problem with that version we can eliminate that possibility. 

(If it is an R version problem it will be easy to find when we switch the CI to GitHub Actions which allows us to test with multiple versions of R)
